src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -578,15 +584,47 @@ public static String getInvalidityInfoForTimeForVisibilityStartAndResultsPublish
                 SESSION_VISIBLE_TIME_FIELD_NAME, RESULTS_VISIBLE_TIME_FIELD_NAME);
     }
 
+    /**
+     * Checks if the session end time is before all extended deadlines.
+     * @return Error string if any deadline in {@code deadlines} is before {@code sessionEnd}
+     *         Empty string if any deadline in {@code deadlines} is after {@code sessionEnd}
```

Comment:
Yes, I would also usually do that too. However, I was unsure about this one because it seemed like they followed the same strange convention within this file and I was unsure if I had to follow it too.
![image](https://user-images.githubusercontent.com/65202977/159129161-2bd30e21-eda6-4d47-81eb-0b693fbb7e7e.png)
This clears my doubt; I will stick to the one-liner then.

Reply:
I think the former is ok. The verbosity is unnecessary to me. We can start from here and slowly change up the rest?